### DISCUSSION TITLE
SAM model in ArcGIS Pro

### DISCUSSION THREAD

**Discussion by ArjunRijal93** [opengeos/segment-geospatial#176] [Q&A] [answered]
I have been attempting to implement the SAM model in ArcGIS Pro, following the guidelines provided on your website, "https://samgeo.gishub.org/examples/arcgis/". Despite successfully installing the deep learning framework, I encountered an error when I ran the sample code you made available. I've attached a screenshot of the error message to review.
![thumbnail_image](https://github.com/opengeos/segment-geospatial/assets/136386072/b2b9e2fd-575c-48b8-a7c2-ae092aa94b02)

**Comment by giswqs** [comment, marked as answer]
Try `mamba install -c conda-forge opencv`

**Reply by ArjunRijal93** [reply]
Did not work

**Reply by giswqs** [reply]
What do you mean it did not work? Try the following code. If it does not work, it is an opencv installation issue, not really a samgeo issue. 

```python
import cv2
```
